etc/Dockerfile

```diff
@@ -36,9 +36,18 @@ ENV LANG en_US.UTF-8
 COPY --from=builder /usr/src/grin/target/release/grin /usr/local/bin/grin
 
 WORKDIR /root/.grin
-RUN grin server config && \
+
+ARG grinnet=""
+
+RUN grin ${grinnet} server config && \
     sed -i -e 's/run_tui = true/run_tui = false/' grin-server.toml
 
+VOLUME ["/root/.grin"]
+
 EXPOSE 13413 13414 13415 13416
 
-ENTRYPOINT ["grin", "server", "run"]
+EXPOSE 3413 3414 3415 3416
+
+ENTRYPOINT ["grin"]
```

Comment:
Shouldn't you be using grinnet here?

Reply:
Not really .. since ARG is only available at buildtime. In run time, passing `--floonet` should just be regarded as extra cli args, which are now supported. So for running a floonet container, I use:
`docker run --rm -it -p 13413-13416:13413-13416 -v dotgrin:/root/.grin --name grin grin --floonet server run`